main.go

```diff
@@ -163,20 +147,82 @@ func main() {
 	}
 }
 
+func prepareOutput(outputSlice []string) (tracee.OutputConfig, error) {
+	outputHelp := `
+Control how and where output is printed.
+Possilbe options:
+
+[format:]{table,table-verbose,json,gob,gotemplate=/path/to/template}   output events in the specified format. for gotemplate, specify the mandatody template file
+out-file:/path/to/file                                                 write the output to a specified file. the path to the file will be created if not existing and the file will be deleted if existing (deafult: stdout)
+err-file:/path/to/file                                                 write the errors to a specified file. the path to the file will be created if not existing and the file will be deleted if existing (deafult: stderr)
+option:{eot,stacktrace,detect-syscall,exec-env}                        augment output according to given options (default: none)
+	eot                                                                add a final event that signals the end of event stream. this is an empty event with "EventName" set to the ASCII code 4, which is "End Of Transmission"
+	stack-addresses                                                    include stack memory addresses for each event
+	detect-syscall                                                     when tracing kernel functions which are not syscalls, detect and show the original syscall that called that function
+	exec-env                                                           when tracing execve/execveat, show the environment variables that were used for execution
+Examples:
+	--output json --output option:eot                        | output as json and add an EOT event
+	--output gotemplate=/path/to/my.tmpl                     | output as the provided go template
+	--output out-file:/my/out err-file:/my/err                 | output to /my/out and errors to /my/err
+
+Use this flag multiple times to choose multiple capture options	
+`
+
+	res := tracee.OutputConfig{}
+	if len(outputSlice) == 1 && outputSlice[0] == "help" {
+		return res, fmt.Errorf(outputHelp)
+	}
+
+	for _, o := range outputSlice {
+		outputParts := strings.SplitN(o, ":", 2)
+		if outputParts[0] == "out-file" {
+			res.OutPath = outputParts[1]
```

Comment:
What if the input "out-file" was given? In that case, there is no index 1 to outputParts